pydeseq2/DeseqDataSet.py

```diff
@@ -266,22 +276,47 @@ def fit_genewise_dispersions(self):
         rough_disps = self._rough_dispersions.loc[self.non_zero_genes].values
         size_factors = self.size_factors.values
 
-        with parallel_backend("loky", inner_max_num_threads=1):
-            mu_hat_ = np.array(
-                Parallel(
+        # mu_hat is initialized differently depending on the number of different factor
```

Comment:
do you think it might be useful to through a warning or otherwise give the comment to the user on which fit was used? Or at least add it to the docstring so it is easier to find? (please ignore if you don't think it is useful information to the user)

Reply:
That's a good question. Personally I don't think a warning is necessary, as the `_mu_hat` attribute is not meant to be accessed by the user (it's an intermediate value). DESeq2 does this silently too. If that's OK for you, I'll leave this as is for now.